packages/lesswrong/components/comments/DebateResponseBlock.tsx

```diff
@@ -154,6 +166,14 @@ export const DebateResponseBlock = ({ responses, post, orderedParticipantList, d
         className={classes.menu}
       />;
 
+      const votingSystemName = comment.votingSystem || "default";
+      const votingSystem = getVotingSystemByName(votingSystemName);
+      const voteProps = useVote(comment, "Comments", votingSystem);
```

Comment:
Given the lint errors, it seems maybe like you'll need to create a new component to render each comment, so you can properly use hooks inside of it.

Reply:
Did it. We don't konw this component still needs to exist though, and it could maybe just be rolled into its parent (whatever that is).